cmake/Modules/OpmStaticTargets.cmake

```diff
@@ -7,15 +7,23 @@
 
 # Macros
 function(opm_from_git repo name revision)
+  if(ARGN)
+    set(COMMANDS BUILD_COMMAND ${CMAKE_COMMAND} --build <BINARY_DIR> --target ${ARGN}
+                 INSTALL_COMMAND)
```

Comment:
we want it to clone the tip of your source tree for opm-upscaling and i don't know what that is. it's a cheap way to just stay at whatever state you get after cloning.